### ISSUE TITLE
Duplicate import when multiple aliases and comment

### ISSUE THREAD

**Issue by ericklaus-wf** [rinchsan/gosimports#69]
When importing the same module under different aliases _and_ when the second import has a comment with a right-paren, the second import is duplicated.

Input file:

```golang
package foo

import (
	"fmt"
	blah "fmt"  // ) fun
)

func something() {
	fmt.Println(`hi`)
	blah.Println(`hi`)
}
```

Actual output:

```golang
package foo

import (
	"fmt"
	blah "fmt"  // ) fun

	blah "fmt"  // ) fun
)

func something() {
	fmt.Println(`hi`)
	blah.Println(`hi`)
}
```

This is a pretty degenerate case! We're triggering this only because we have a tool in our repo that detects imports aliased under different names and gosimports is formatting its test file.

**Comment by uhthomas** [comment]
I also noticed this, but under different circumstances. The following snippet

```go
// <some C stuff>
import "C"

import (
    "context"
    "fmt"
)
```

becomes

```go
// <some C stuff>
import "C"

import (
    "C"
    "context"
    "fmt"
)
```

**Comment by rinchsan** [comment]
Thank you guys for the contributions to report issues!

Issue reported by @ericklaus-wf has been resolved by @shuntagami #77 .
Issue reported by @uhthomas has been resolved by #98 .

And a new version of gosimports has been released, including fixes above.
https://github.com/rinchsan/gosimports/releases/tag/v0.2.1

Closing this one.
